packages/BackToTopButton/index.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <button
+    :class="[$style.float, { show: isShown }]"
+    @click="scrollToTop"
+    v-html="require('~/assets/ic-arror-top.svg')"
+  />
+</template>
+
+<script>
+export default {
+  name: 'BackToTopButton',
+  data() {
+    return {
+      isShown: false
+    }
+  },
+  created() {
+    this.handleScroll()
+    window.addEventListener('scroll', this.handleScroll)
+  },
+  destroyed() {
+    window.removeEventListener('scroll', this.handleScroll)
+  },
+  methods: {
+    handleScroll() {
+      const { innerHeight, pageYOffset } = window
+      this.isShown = pageYOffset > innerHeight
+    },
+    scrollToTop() {
+      window.scrollTo({
+        top: 0,
+        left: 0,
+        behavior: 'smooth'
+      })
+    }
+  }
+}
+</script>
+
+<style lang="scss" module>
+@import '~/styles/_variables.scss';
+@import '~/styles/extend/flexbox.scss';
+@import '~/styles/extend/reset.scss';
+
+.float {
+  @extend %flex-center;
+  @extend %reset-button;
+
+  position: fixed;
+  background-color: $white;
+  right: $spacing-1;
+  bottom: $spacing-2;
+  width: $fs-36;
+  height: $fs-36;
```

Comment:
I think `$fs-xx` is for text size, it does not make sense to put it with `width` and `height`.

@Bruce-Lan

Reply:
Personally prefer to set the width / height with `rem`, so the layout would change accordingly  to the font-size changes by the browser setting. Currently we're only using rem for font sizes, but using them to adjust layout is more responsive IMO.

Reply:
If @Bruce-Lan insist to keep the values to specific `px` setting, it's okay for me to change them back

Reply:
I have no doubt about the `rem` usage, just wondering the variable definition, maybe we can ask @Bruce-Lan about this.

Reply:
Oh I Got it, it's just seems to be misleading to use `fs` as `width`. I'd prefer defining something like `$unit-16` to describe both (??). 
I'll use `rem` directly for now anyway 👌